src/pages/Organizations/EditOrganization.tsx

```diff
@@ -73,56 +81,91 @@ const EditOrganization = () => {
+  );
+};
+
+export const EditOrganization = () => {
+  const [state, actions] = useContext(Context);
+  const [emailAddress, setEmailAddress] = useState<string>("");
+  const { currentOrganization } = useParams<OrgInterface>();
+  const [filteredOrg] =
+    state.user.orgs?.filter((org) => org.name === currentOrganization) ?? [];
+
+  const handleSubmit = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    actions.inviteToOrg(emailAddress, currentOrganization, (_invite, err) => {
+      console.log(_invite);
+      if (err) {
+        console.error(err);
+        return;
+      }
+    });
+  };
+  return (
+    <EditOrganizationContainer>
+      <h1>Edit Organization</h1>
+      <hr />
       <OrganizationOptions>
         <Panel>
           <h2>General Information</h2>
-          <FormInput name="orgName" type="text" label="Organization Name" />
-          <FormInput name="orgName" type="text" label="Description" disabled />
+          {/* <FormInput
+            name="orgName"
+            type="text"
+            label="Organization Name"
+            value={filteredOrg.name}
+          /> */}
+          {filteredOrg && filteredOrg.name}
+          {/* <FormInput name="orgName" type="text" label="Description" disabled /> */}
           <h3>Add Member</h3>
           <p>Manage Team Members edit team information and permissions</p>
-          <FormInput name="newMember" type="text" label="Email Address" />
-          <PrimaryButton>Send Invite</PrimaryButton>
+          <form onSubmit={handleSubmit}>
+            <FormInput
+              name="newMember"
+              type="text"
+              label="Email Address"
+              value={emailAddress}
+              onChange={(e) => setEmailAddress(e.target.value)}
+            />
+            <PrimaryButton>Send Invite</PrimaryButton>
+          </form>
         </Panel>
         <RightPanel>
           <h3>Current Members</h3>
           <MemberList>
-            <thead>
-              <tr>
-                <th>name</th>
-                <th>actions</th>
-              </tr>
-            </thead>
-            <tbody>
-              <tr>
-                <td>email-malio@email.com</td>
-                <td>
-                  <TertiarySmallButton>Remove</TertiarySmallButton>
-                </td>
-              </tr>
-              <tr>
-                <td>email-malio@email.com</td>
-                <td>
-                  <TertiarySmallButton>Remove</TertiarySmallButton>
-                </td>
-              </tr>
-              <tr>
-                <td>email-malio@email.com</td>
-                <td>
-                  <TertiarySmallButton>Remove</TertiarySmallButton>
-                </td>
-              </tr>
-              <tr>
-                <td>email-malio@email.com</td>
-                <td>
-                  <TertiarySmallButton>Remove</TertiarySmallButton>
-                </td>
-              </tr>
-            </tbody>
+            {filteredOrg && (
```

Comment:
table here, list on the overview page... oopts, took me ages to realize that :P

Reply:
note: there might be some dom conflicts because members list (the name) is used for two diff components?